src/Akka.Persistence.SqlServer.Tests/Batching/BatchingSqlServerCurrentPersistenceIdsSpec.cs

```diff
@@ -43,9 +43,9 @@ class = ""Akka.Persistence.SqlServer.Journal.BatchingSqlServerJournal, Akka.Pers
             return conf.WithFallback(SqlReadJournal.DefaultConfiguration());
         }
 
-        protected override void Dispose(bool disposing)
```

Comment:
TCK supposedly does not support disposal for Akka.Persistence.Query specs, which would be a breaking change from before - this one caught me by surprise. CC @Arkatufus

Reply:
There's also no more `base.Dispose` method.